Package.swift

```diff
@@ -16,7 +16,7 @@ let package = Package(
         .package(url: "https://github.com/kylef/Spectre.git", from: "0.9.0"),
         .package(url: "https://github.com/onevcat/Rainbow.git", from: "3.0.0"),
         .package(url: "https://github.com/tuist/xcodeproj.git", .exact("7.1.0")),
-        .package(url: "https://github.com/jakeheis/SwiftCLI.git", .exact("5.2.2")),
+        .package(url: "https://github.com/jakeheis/SwiftCLI.git", .exact("5.3.2")),
```

Comment:
We can make this `from`, incase other packages using XcodeGen also use SwiftCLI

Reply:
I prefer to use `upToNextMinor(from:)` than `from`. Because SwiftCLI brought breaking change when minor versions are updated.